MvcApp/MvcApp/Services/Interfaces/IAdminPostServices.cs

```diff
@@ -0,0 +1,15 @@
+﻿using MvcApp.Models;
+
+namespace MvcApp.Services.Interfaces
+{
+    public interface IAdminPostServices
+    {
+        public List<Post> GetAll();
```

Comment:
public dib yozish shart emas. interfaceni methodlari shundoq ham public bo'ladi.

boshqa interfacelarda ham to'g'irlab chiqin

Reply:
togriladim